### DISCUSSION TITLE
can't run Diorisis Search 3.1

### DISCUSSION THREAD

**Discussion by carvalhar** [bottlesdevs/Bottles#294] [General]
Hi,

I have Bottles working fine on Endless OS (installed with FlatPack), but i can't install Diorisis Search 3.1 (https://www.crs.rm.it/diorisissearch/) using Wine/Bottles (linux app doesn't work). It also doesn't give me any error. 
What should I do?

Than k you

**Comment by mirkobrombin** [comment]
It looks more like a Wine problem than Bottles (which is just a wine prefix manager). Anyway I would need a log or error to help you even if he doesn't give it to you.

Some applications are not compatible with Wine, maybe it is one of them. I suggest you try to install other runners (eg Proton or Lutris) from the preferences and create a custom bottle with one of these.

**Reply by carvalhar** [reply]
How can i get a log? I tried to open the debug window, but nothing is printed there, as no error occured.

**Reply by carvalhar** [reply]
I tried chadornay, proton and Lutris and it doesn't work. Lutris required some .NET package to install, but after this, nothing happened again;

**Reply by mirkobrombin** [reply]
> How can i get a log? I tried to open the debug window, but nothing is printed there, as no error occured.

Run Bottles through the terminal.



> I tried chadornay, proton and Lutris and it doesn't work. Lutris required some .NET package to install, but after this, nothing happened again;

Surely that program is not compatible with Wine. Try asking on the WineHQ forum.
